using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -103,116 +119,150 @@ This API synchronizes the provided customer identifier type key and value with t
+
+#### Examples
 
 **Objective-C**
 
 ```objectivec
-NSDictionary *ids = @{@"idType":@"idValue"};
-[ACPIdentity syncIdentifiers:ids authentication:ACPMobileVisitorAuthenticationStateAuthenticated];
+NSDictionary *ids = @{@"idType1":@"idValue1", 
+                      @"idType2":@"idValue2", 
+                      @"idType3":@"idValue3"};
+[ACPIdentity syncIdentifiers:ids];
 ```
 
 **Swift**
 
 ```swift
-let identifiers : [String: String] = ["idType1":"idValue1", "idType2":"idValue2"];ACPIdentity.syncIdentifiers(identifiers, authentication:
-ACPMobileVisitorAuthenticationState.authenticated)
+let identifiers : [String: String] = ["idType1":"idValue1",
+                                      "idType2":"idValue2",
+                                      "idType3":"idValue3"];
+ACPIdentity.syncIdentifiers(identifiers)
 ```
+
 {% endtab %}
 
 {% tab title="React Native" %}
 #### JavaScript
 
-### syncIdentifier
+### syncIdentifier<a id="syncIdentifier-js"></a>
 
-Updates the provided customer ID with the Adobe Experience Cloud ID Service.
+This `syncIdentifier()` API updates or appends the provided customer identifier type key and value with the given authentication state to the Adobe Experience Cloud ID Service. If the given customer ID type already exists in the service, then it is updated with the new ID and authentication state. Otherwise a new customer ID is added.
 
-This API synchronizes the provided customer identifier type key and value with the provided [authentication state](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#authenticationstate) to the Adobe Experience Cloud ID Service. If this customer ID type exists in the service, this type is updated with the new ID and authentication state. Otherwise a new customer ID is added. This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
+#### Syntax
 
 ```jsx
-ACPIdentity.syncIdentifier(identifierType, identifier, authenticationState);
+ACPIdentity.syncIdentifier(identifierType: String, identifier: String, authenticationState: string)
 ```
 
-### syncIdentifiers
+o *identifierType* (String) containing identifier type, and the *identifier* (String) containing identifier value; should not be null or empty. If the identifierTtype and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+o  *authenticationState* (VisitorIDAuthenticationState) value indicating authentication state for the user contaisn one of the VisitorID.AuthenticationState values `ACPMobileVisitorAuthenticationState.AUTHENTICATED`,`ACPMobileVisitorAuthenticationState.LOGGED_OUT` and `ACPMobileVisitorAuthenticationState.UNKNOWN`
```

Comment:
Please reword as follows:

*authenticationState* (VisitorIDAuthenticationState) indicates the authentication state for the user and contains one of the following VisitorID.AuthenticationState values:

  * `ACPMobileVisitorAuthenticationState.AUTHENTICATED`
  * `ACPMobileVisitorAuthenticationState.LOGGED_OUT`
  * `ACPMobileVisitorAuthenticationState.UNKNOWN`